static/js/redux/ui/advising.jsx

```diff
@@ -82,6 +82,7 @@ class Advising extends React.Component {
   
   fetchSemesters() {
     let semesters = [ this.props.semester.current ]; //the current semester 
+    console.log(semesters); // TODO: figure out why this currently undefined?
```

Comment:
you need to pass in semester as a prop!

Reply:
at the bottom where you have `Advising.propTypes` add a `semester` prop.

Reply:
I just updated literally few seconds ago, can you check again.

Reply:
```
semester: PropTypes.shape({
    name: PropTypes.string.isRequired,
    year: PropTypes.string.isRequired,
  }).isRequired,
```